JS/app.js

```diff
@@ -18,7 +18,7 @@ Project.prototype.toHtml = function(){
   $newProject.find('address').html(this.author);
   $newProject.find('address a').attr('href', this.projectUrl);
   $newProject.find('h1').html(this.title);
-  $newProject.find('h1').append('<h6>' + this.category + '</h6>');
+  $newProject.find('h1').append('<h5>' + this.category + '</h5>');
```

Comment:
I notice that you have `h5` tags here, but you don't have `h5` styled in your CSS. Is that deliberate?